examples/plot_connectivity.py

```diff
@@ -50,15 +50,15 @@
 
 ###############################################################################
 # We can modify the connectivity list to test the effect of different
-# connectivity patterns. For example, we can remove all layer 2 inhibitory
-# connections. In the default network, the src_gids of each connection are
-# all the same cell type.. Connections are stored under ``conn['gid_pairs']``
-# as a dictionary indexed by src_gid:
-# ``{src_gid1: [target_gid1, target_gid2], ...]``. Each src_gid indexes a
-# list with its target gids
-new_connectivity = [conn for conn in net.connectivity
-                    if conn['src_type'] != 'L2_basket']
-net.connectivity = new_connectivity
+# connectivity patterns. For example, we can remove 90% all layer 2 inhibitory
+# connections.
+# This can be achieved with :meth:`hnn_core._Connectivity.update_probability`,
+# a function that is available for each element of ``net.connectivity``.
+# Note that in the default network, the src_gids of each connection are
+# all the same cell type allowing for easy modifications.
+for conn in net.connectivity:
+    if conn['src_type'] == 'L2_basket':
+        conn.update_probability(0.1)
```

Comment:
Can we use the `net.add_connection(..., probability)` method so we can still play around with `_Connectivity` class. I suspect we'll need to make changes when we unify it with the drive connectivity